djangocms_fil_permissions/permissions.py

```diff
@@ -0,0 +1,40 @@
+from django.core.exceptions import PermissionDenied
+
+from rules.rulesets import RuleSet
+
+from .rules import has_site_access
+
+
+site_permissions = RuleSet()
+site_permissions.add_rule("site_perm", has_site_access)
+
+
+class SitePermissionBackend(object):
+    """Authentication backend that checks row-level permissions granted
+    on site-level.
+    """
+
+    def authenticate(self, request, **credentials):
```

Comment:
Why does this return None? We should document design decisions please.

Reply:
Standard django behaviour: https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#writing-an-authentication-backend